_includes/com-ingest/com-ingest-csv-header-flag-tls.md

```diff
@@ -1,7 +1,11 @@
 ### Connect securely over TLS and define header flags
 
 Use this method to:
+<<<<<<< HEAD
```

Comment:
needs cleanup

Reply:
Hell, I didn't notice that. Thanks for letting me know